pkgs/development/tools/build-managers/bazel/common.nix

```diff
@@ -0,0 +1,563 @@
+{ version
+, sha256
+, src_deps
+, javac11
```

Comment:
Is `javac11` really the correct name here? Perhaps `javac11_version` is better here?

Reply:
Should this also be camelCase?

Reply:
Yeah, I think for consistency that would be good. Usually in these cases we also group the attributes with a small header saying what they are. Look at pkgs/applications/networking/browsers/firefox/common.nix for an example.

Reply:
Resolved?